http.go

```diff
@@ -12,31 +14,18 @@ import (
 	"net/url"
 	"os"
 	"path/filepath"
-	"regexp"
+	"sort"
 	"strings"
 	"unicode"
 
 	"code.google.com/p/go.net/publicsuffix"
+	"github.com/juju/loggo"
 	"github.com/juju/persistent-cookiejar"
 	"gopkg.in/macaroon-bakery.v0/httpbakery"
 	flag "launchpad.net/gnuflag"
 	"launchpad.net/rjson"
 )
 
-var flags struct {
-	json       bool
-	form       bool
-	headers    bool
-	body       bool
-	rjson      bool
-	raw        bool
-	noBrowser  bool
-	basicAuth  string
-	cookieFile string
-
-	// TODO auth, verify, proxy, file, timeout
-}
-
```

Comment:
Not really, but I'll probably change the examples in a future PR, when we've got a better name for this command.